dubbo-plugin/dubbo-qos/src/main/java/org/apache/dubbo/qos/command/util/ServiceCheckUtils.java

```diff
@@ -59,4 +59,24 @@ public static int getConsumerAddressNum(ConsumerModel consumerModel) {
         }
         return num;
     }
+
+    //for the situation that QoS wants to query the address number of a consumer in all Registries
+    public static int getConsumerAddressNumInAllRegistries(ConsumerModel consumerModel){
```

Comment:
Is there any way to call this method?

Reply:
![image](https://user-images.githubusercontent.com/18096538/134862174-8d780eec-e756-425e-bfd1-9eaed1095e12.png)
I raised this issue inspired by this "TODO" comment, thinking about there is some conditions that we need to check all registries.
![image](https://user-images.githubusercontent.com/18096538/134862784-1b67ced7-6ad6-4bba-ad96-782806248848.png)
And this "getConsumerAddressNum" method is called two times in the whole project. So I think it makes sense that I added a equal-functional method which is for checking all registry here. But I don't know how to call this method meaningfully at the moment.
Anyway, if you guys think this method is redundant in version 2.7.15, feel free to close this issue. Hope Dubbo can be better and better.